R/subset_wuse.R

```diff
@@ -17,13 +17,13 @@
 #' areas <- "10" # NA uses all areas
 #' area.column <- "STATECODE"
 #' 
-#' w.wuse.sub <- subset_wuse(w.use, data.elements,area.column,areas)
+#' w.use.sub <- subset_wuse(w.use, data.elements,area.column,areas)
 subset_wuse <- function(w.use, data.elements, area.column, areas=NA) {
   # subset w.use on basis of areas and select the data elements of interest
   if (all(is.na(areas))){
-    w.use.sub <- w.use[,c("YEAR",area.column,data.elements)]
+    w.use.sub <- w.use[,c("YEAR","STATECOUNTYCODE",area.column,data.elements)]
```

Comment:
My assumption was that `w.use.sub` could work with any data set for nearly all the functions, and I guess there's no easy (?) way to get the key you need for the huc datasets and the excel "State Compiler data set"? My vote would be to make that a separate function...but if you want to use this function...make sure you test it with all the different kinds of data. I'll hopefully be pushing out my pr very soon....there are a lot of improvements to the get_awuds_data function that  should work on all the data jenn gave us.